src/routes/reference/solid-router/data-apis/create-async.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: createAsync
+---
+
+This is light wrapper over [`createResource`](/reference/create-resource) that aims to serve as stand-in for a future primitive we intend to bring to Solid core in 2.0. It is a simpler async primitive where the function tracks like `createMemo` and it expects a promise back that it turns into a Signal. Reading it before it is ready causes Suspense/Transitions to trigger.
+
+```jsx
+const user = createAsync(() => getUser(params.id));
+```
+
+<Callout>
+	Using `cache` in `createResource` directly won't work properly as the fetcher
+	is not reactive and it won't invalidate properly.
+</Callout>
```

Comment:
I might be missing something but what does this comment have to do with `createAsync`?